lib/src/fonts.dart

```diff
@@ -1,101 +1,88 @@
 import "package:flutter/material.dart";
 
+/// Pixels per point constant.
+/// In CSS:
+/// pt = 1/72 inch
+/// px = 1/96 inch
+const double pixelsPerPoint = 96.0 / 72.0; // 1.333(3)
+
 /// The text fonts supported on aRdent smart glasses
 @immutable
 class GYWFont {
-  /// The name of the font
-  final String name;
-
-  /// The internal prefix of the font
-  final String prefix;
+  /// The font filename on the GYW device (5 characters long and no type extension).
+  final String filename;
 
-  /// The size (in points) of a character
-  final int size;
+  /// The name of the font.
+  final String name;
 
-  /// The width (in pixels) of a character
-  final int width;
+  /// The font family name.
+  final String fontFamily;
 
-  /// The height (in pixels) of a character
-  final int height;
+  /// The width of a character at 1pt.
+  final double charWidth;
 
-  /// Whether the fontweight is bold (w700) or normal (w400)
+  /// Whether the fontweight is bold (w700) or normal (w400).
   final bool bold;
 
+  /// Whether the font style is italic.
+  final bool italic;
+
   /// Creates a font.
   const GYWFont({
-    required this.name,
-    required this.prefix,
-    required this.size,
-    required this.width,
-    required this.height,
-    required this.bold,
-  });
+    required this.filename,
+    this.name = "Custom Font",
+    this.fontFamily = "Custom Font",
```

Comment:
I've thought at something regarding these two. What do you think of having them nullable, and then null by default ?

Reply:
Yes, that's better. Optionality should be conveyed throurgh nullability.